python/ray/llm/tests/serve/cpu/configs/test_models.py

```diff
@@ -275,5 +276,64 @@ def test_experimental_configs(self):
             )
 
 
+class TestPublicLLMConfig:
```

Comment:
This test should actually live in `serve/ ` we already have one test file there that tests LLMConfig can get imported.